tepeszCodes/document-management-system
# Chore: Implement barrel pattern

#### What does this PR do?
Implements barrel pattern in our project
#### How should this be manually tested?
There is no other method than to look at the files and check if they are structured correctly and if imports / exports actually import / export what they should or not
